src/sparsezoo/utils/download.py

```diff
@@ -93,13 +93,24 @@ def __init__(
         self.chunk_bytes = chunk_bytes
         self.job_queues = Queue()
         self._lock = threading.Lock()
-        self.chunk_folder = self.get_parent_chunk_folder(download_path)
-
-    def get_parent_chunk_folder(self, path: str) -> str:
-        """Get the name of the file that is used as folder inside chunks"""
-        path = path.split(os.path.sep)[-1]
-        path = path.replace(".", "_")
-        return path
+        self.chunk_download_path = self.get_chunk_download_path(download_path)
+
+    def get_chunk_download_path(self, path: str) -> str:
+        """Get the path where chunks will be downloaded"""
+
+        # make the folder name from the model name and file to be downloaded
+        stub = path.split(os.path.sep)[-3]
+        path = "_".join(path.split(os.path.sep)[-2:])
+        file_name_as_folder = path.replace(".", "_")
+
+        # save the chunks on a different folder than the root model folder
+        return os.path.join(
+            os.path.abspath(
+                os.path.expanduser("~/.cache/sparsezoo/neuralmagic/chunks")
```

Comment:
Can we get this from a constant elsewhere? ~ is going to be OS dependent I believe